charts/tidb-operator/values.yaml

```diff
@@ -185,30 +185,27 @@ admissionWebhook:
   ## if admissionWebhook.create and admissionWebhook.hooksEnabled.pods are both enabled,
   ## The pre-delete-job would delete the validationWebhookConfiguration using this image
   jobImage: "bitnami/kubectl:latest"
-  hooksEnabled:
+  ## validation webhook would check the given request for the specific resource and operation
+  validation:
     ## statefulsets hook would check requests for updating tidbcluster's statefulsets
     ## If enabled it, the statefulsets of tidbcluseter would update in partition by tidbcluster's annotation
     statefulSets: false
     ## pods hook would check requests for creating and deleting tidbcluster's pods
     ## if enabled it, the pods of tidbcluster would safely created or deleted by webhook instead of controller
     pods: true
     ## validating hook validates the correctness of the resources under pingcap.com group
-    validating: false
+    pingcapResources: false
```

Comment:
Do we support `true` now? If yes, why set it to `false` by default?

Reply:
I guess the resource validating is not verified yet.